### PR TITLE
Update widget-tester and rv-common-e2e

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Update widget-tester and rv-common-e2e

[stage-19]

## Motivation and Context
Vulnerability fixes for those repos.

## How Has This Been Tested?
Build passes. Coverage is updated. Staging works.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc @santiagonoguez please review. Thanks!

File: package.json [hunk 1]
```diff
@@ -41,8 +41,8 @@
     "jshint": "^2.10.3",
     "jshint-stylish": "^2.0.1",
     "run-sequence": "^1.1.1",
-    "rv-common-e2e": "git://github.com/Rise-Vision/rv-common-e2e.git",
-    "widget-tester": "git://github.com/Rise-Vision/widget-tester.git"
+    "rv-common-e2e": "git://github.com/Rise-Vision/rv-common-e2e.git#dependabot/npm_and_yarn/ini-1.3.8",
+    "widget-tester": "git://github.com/Rise-Vision/widget-tester.git#fix/vulnerabilities"
```
Comment (santiagonoguez) [inline package.json:45]: you'll update to point to the release number, right ?

Comment (alex-deaconu) [reply package.json:45]: @santiagonoguez well, yeah, I will clear the release so it uses the latest tag. Thanks.

**Comment by alex-deaconu** [comment]
@santiagonoguez latest commit reverts both tags from package.json and only keeps the package-lock.json changes. Thanks.

**Review by santiagonoguez — APPROVED** [review]

**Pull request merged**